probeinterface/plotting.py

```diff
@@ -118,19 +119,21 @@ def plot_probe(probe, ax=None, contacts_colors=None,
 
     if title:
         ax.set_title(probe.get_title())
-    
+
     return poly, poly_contour
 
-def plot_probe_group(probegroup, same_axe=True, **kargs):
+def plot_probe_group(probegroup, same_axis=True, **kargs):
```

Comment:
No. In matplotlib terminology
axe =  the plot
axis (yaxis or yaxis) = the line to handle ticks.
I think this is more relevant to keep axe.

Maybe `same_ax` would be the less confusing. COudl you change this ?

Reply:
I think `same_axes` would be correct in this case. "Axes" is the plural of "axis" (whereas "axe" is a tool for chopping wood). I updated this in the latest set of commits.

Reply:
OK. I am stupid. Thank for this.